target/linux/ramips/dts/rt5350_olimex_rt5350f-olinuxino-evb-plus.dts

```diff
@@ -0,0 +1,119 @@
+/dts-v1/;
+
+#include "rt5350_olimex_rt5350f-olinuxino-evb.dts"
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "olimex,rt5350f-olinuxino-evb-plus", "ralink,rt5350-soc";
+	model = "Olimex RT5350F-OLinuXino-EVB+";
+
+	aliases {
+		spi0 = &spi_mmc;
+		spi1 = &spi_uext;
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		acin {
+			gpio-export,name = "acin";
+			gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	spi_mmc: spi-mmc {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		ranges;
+
+		gpio-sck = <&gpio0 4 GPIO_ACTIVE_HIGH>;
+		gpio-miso = <&gpio0 6 GPIO_ACTIVE_HIGH>;
+		gpio-mosi = <&gpio0 5 GPIO_ACTIVE_HIGH>;
```

Comment:
These definitions change in newer kernels:
https://www.kernel.org/doc/Documentation/devicetree/bindings/spi/spi-gpio.txt

Since we will bump this target to 4.19 soon, I recommend adding the new properties right away additional to the old ones:

```
		sck-gpios = <&gpio0 4 GPIO_ACTIVE_HIGH>;
		miso-gpios = <&gpio0 6 GPIO_ACTIVE_HIGH>;
		mosi-gpios = <&gpio0 5 GPIO_ACTIVE_HIGH>;

		/* remove on kernel 4.19 */
		gpio-sck = <&gpio0 4 GPIO_ACTIVE_HIGH>;
		gpio-miso = <&gpio0 6 GPIO_ACTIVE_HIGH>;
		gpio-mosi = <&gpio0 5 GPIO_ACTIVE_HIGH>;
```